### PR TITLE
chore: added an idea and project

### PR DESCRIPTION
**Pull request by emily876**
added an idea and project in JSON files

### CODE REVIEW

File: data/projects.json [hunk 1]
```diff
@@ -138,6 +138,16 @@
...
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Venessa-My-Personal-Assistant",
+        "display": "Personal-Assitant using python",
+        "description": "It will assist you with simple tasks like sending an email with just on your command and greets you too and can help you search for anything on google , can play music, tell the current time, and so on..",
+        "author": "emily876",
+        "category": "desktop application",
+        "level": "basic",
+        "site": "https://github.com/emily876/Venessa-My-Personal-assistant",
```
Comment (ming-tsai) [inline data/projects.json:149]: This attribute is only for websites.

File: data/projects.json [hunk 2]
```diff
@@ -138,6 +138,16 @@
         "level": "basic",
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Venessa-My-Personal-Assistant",
+        "display": "Personal-Assitant using python",
```
Comment (ming-tsai) [inline data/projects.json:144]: The display name could be 
```suggestion
        "display": "Personal Assitant using Python",
```

File: data/ideas.json [hunk 3]
```diff
@@ -40,5 +40,11 @@
         "description": "Create a rock paper scissors game! Try recreating the classic game, or put your own fun twist on it.",
         "level": "basic",
         "tags": "rock-paper-scissors, game, classic"
+    },
+    {
+        "title": "A personal assistant for your system using python",
```
Comment (ming-tsai) [inline data/ideas.json:45]: Could you remove the programming language name? due to this could be programming on another language

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @emily876,
Just a couple comments

File: data/ideas.json (hunk 3 above)
Comment (emily876) [reply data/ideas.json:45]: Thanks for reviewing, and surely I will do all the changes as required.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**